src/patternfly/components/Truncate/examples/Truncate.md

```diff
@@ -14,27 +14,55 @@ The truncate component contains two child elements, `.pf-v5-c-truncate__start` a
 ### Default
 ```hbs
 <div class="pf-v5-c-truncate--example">
-  {{#> truncate truncate--id="default-truncation-example"}}
-    {{> truncate-start truncate-start--text="Vestibulum interdum risus et enim faucibus, sit amet molestie est accumsan."}}
+  {{#> truncate truncate--id='default-truncation-example'}}
+    {{> truncate-start truncate-start--text='Vestibulum interdum risus et enim faucibus, sit amet molestie est accumsan.'}}
   {{/truncate}}
 </div>
 ```
 
 ### Middle
 ```hbs
 <div class="pf-v5-c-truncate--example">
-  {{#> truncate truncate--id="middle-of-line-truncation-example"}}
-    {{> truncate-start truncate-start--text="redhat_logo_black_and_white_reversed_simple_with_fedora_con"}}
-    {{> truncate-end truncate-end--text="tainer.zip"}}
+  {{#> truncate truncate--id='middle-of-line-truncation-example'}}
+    {{> truncate-start truncate-start--text='Vestibulum interdum risus et enim faucibus,&nbsp;'}}
+    {{> truncate-end truncate-end--text='sit amet molestie est accumsan.'}}
   {{/truncate}}
 </div>
 ```
 
 ### Start
 ```hbs
 <div class="pf-v5-c-truncate--example">
-  {{#> truncate truncate--id="start-truncation-example"}}
-    {{> truncate-end truncate-end--text="Vestibulum interdum risus et enim faucibus, sit amet molestie est accumsan.&lrm;"}}
+  {{#> truncate truncate--id='start-truncation-example'}}
+    {{> truncate-end truncate-end--text='Vestibulum interdum risus et enim faucibus, sit amet molestie est accumsan.&lrm;'}}
+  {{/truncate}}
+</div>
+```
+
+### File extension
+```hbs
+<div class="pf-v5-c-truncate--example">
+  {{#> truncate truncate--id='default-truncation-file-extension-example'}}
+    {{> truncate-start truncate-start--text='redhat_logo_black_and_white_reversed' truncate-file-extension--text='zip'}}
+  {{/truncate}}
+</div>
+```
+
+### File extension middle
+```hbs
+<div class="pf-v5-c-truncate--example">
+  {{#> truncate truncate--id='middle-of-line-truncation-file-extension-example'}}
+    {{> truncate-start truncate-start--text='redhat_logo_black_and_white_reversed'}}
+    {{> truncate-end truncate-end--text='white_reversed' truncate-file-extension--text='zip'}}
```

Comment:
We'll just need to remove one of the "white_reversed" instances, believe it'd be this one:

```suggestion
    {{> truncate-start truncate-start--text='redhat_logo_black_and_'}}
    {{> truncate-end truncate-end--text='white_reversed' truncate-file-extension--text='zip'}}
```